_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,159 @@
+
+* ELT over ETL
+* SQL based analytics is the way forward
+* Use Technology To Replace Labor Whenever Possible (*So your people can focus on business value instead*)
+
+It's rare to see a book presented in such an accessible way that manages to balance complexity and insight - I recommend it to anyone looking to re-evaluate how they maximise return on investment from their data.
+
+This blog post is in two parts; in this article we will elaborate on modern Cloud Data Warehouses; in the second part we will focus on how their features allow a more flexible and agile way of delivering analytics engineering, as described in the Analytics Guide Book.
+
+## The Cloud Data Warehouse
+
+
+A Data Warehouse is an **enabling technology** for your business to **make decisions** in relation to **the data you gather**. By arranging, conforming, and curating your data to radiate the key performance indicators of your business, a Data Warehouse forms one half of *the presentation layer* - alongside visualisation tools - that enable users in your organisation to identify insights. At least, this is the **traditional view**.
+
+Modern Cloud Data Warehouses can support the existing status quo, including various data modelling methodologies such as Kimball, Inmon or Data Vault for example. Additionally, sticking the word ***cloud*** in front of *data warehouse* does what you would expect cloud to do, in addressing many of the technological limitations or complications associated with traditional Data Warehouses - namely storage availability, compute scalability, and high availability.
+
+However, modern Cloud Data Warehouse offerings typically go far beyond that. Whilst delivering the core competencies of what a Data Warehouse should offer, they often extend these capabilities with features not readily seen before. For example,
+
+* [BigQuery ML](https://cloud.google.com/bigquery-ml/docs/introduction) allows users to inference with Machine Learning models against their datasets directly, using SQL. A more generic feature is also available on Snowflake using [External Functions](https://docs.snowflake.com/en/sql-reference/external-functions-introduction.html) 
+* Snowflake's [Zero Copy Data Sharing](https://docs.snowflake.com/en/user-guide-data-share.html) across accounts in the same cloud region, and [Replication Across Cloud Platforms](https://docs.snowflake.com/en/user-guide/database-replication-failover.html) as described in our recent blog post on [Snowflake Organizations](https://mechanicalrock.github.io/2020/09/23/snowflake-org-structures.html), are a powerful enabler for complex organisations, especially those navigating multi cloud strategies
```

Comment:
You've mixed the spelling of organisations with a Z and S in this line (you've used both). Probably best to pick one. I know Matt is generally in favour of American English

Reply:
I like English English

Reply:
This is a quirky one. "Snowflake Organizations" are a feature so I am treating as a proper noun - I leave their spelling to be consistent with previous articles I've written and also to be consistent with their docs.

For the rest I try to default to Australian English. I will leave this one as is